editor/plugins/script_text_editor.cpp

```diff
@@ -953,13 +953,26 @@ void ScriptTextEditor::_edit_option(int p_op) {
 				if (tx->get_selection_to_column() == 0)
 					end -= 1;
 
+				// Check if all lines in the selected block are commented
+				bool is_commented = true;
+				for (int i = begin; i <= end; i++) {
+					if (!tx->get_line(i).begins_with("#")) {
```

Comment:
What if the line doesn't begin with `#` but it has the comment later on? Example:

```
Not a comment
    # Comment
```

Reply:
damn, forgot to strip it. Thnks, i'll update it.